pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -20,6 +20,7 @@ import (
 	"errors"
 	"fmt"
 
+	scmeta "github.com/kubernetes-incubator/service-catalog/pkg/api/meta"
 	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 	"github.com/kubernetes-incubator/service-catalog/pkg/registry/servicecatalog/server"
 	"github.com/kubernetes-incubator/service-catalog/pkg/storage/tpr"
```

Comment:
Not 100% following what's going on here. Is this the mainline storage code? Is tpr leaking in somehow?